polyfills/Error/isError/detect.js

```diff
@@ -0,0 +1 @@
+"isError" in Error;
```

Comment:
```suggestion
"isError" in Error && "DOMException" in self && Error.isError(new DOMException('error'));
```

Safari seems to have shipped only partial support